lambdas/preview/test/test_index.py

```diff
@@ -68,7 +68,12 @@ def test_fcs(self):
         extended = False
         if (
                 set(os.path.split(f)[1] for f in fcs_files)
-                != set(['accuri-ao1.fcs', 'bad.fcs', '3215apc 100004.fcs'])
+                != {
```

Comment:
that test failed because one of the files is now parsed correctly by newer fcsparser
probably test should be reworked but I decided to add just another file to not deal with it now